src/tests/utils/ApiConstants.ts

```diff
@@ -2,5 +2,6 @@ export const constants = {
 	DEV_CRI_CIC_API_URL: process.env.DEV_CRI_CIC_API_URL?.replace(/\/$/, ""),
 	DEV_IPV_STUB_URL: process.env.DEV_IPV_STUB_URL,
 	DEV_CIC_TEST_HARNESS_URL: process.env.DEV_CIC_TEST_HARNESS_URL,
+	DEV_CIC_SESSION_TABLE_NAME: "session-cic-cri-ddb",
 	DEV_F2F_SESSION_TABLE_NAME: "session-f2f-cri-ddb",
```

Comment:
Can we please remove DEV_F2F_SESSION_TABLE_NAME? It's not being used and also can't be used as we don't have an F2F table in the CIC account